v-next/hardhat-utils/README.md

```diff
@@ -1,3 +1,3 @@
 # Hardhat utils
 
-This package contains utilities used by Hardhat and its plugins.
+This package contains utilities used by Hardhat 3 and its plugins.
```

Comment:
Should we add a comment about it being considered internal as well?

Reply:
done